operator/operator_test.go

```diff
@@ -138,7 +138,7 @@ func initEnvironment(t *testing.T) *environment {
 		if err != nil {
 			panic(err)
 		}
-		time.Sleep(5 * time.Second)
+		time.Sleep(10 * time.Second)
```

Comment:
Instead of a blind sleep, we could run a loop here that tries to connect to the ganache in, say .2 sec steps, for 10 sec. Would speed up the test in most cases. Or maybe instead of a loop, just one request with a 10 sec timeout, not sure what works best.

You could just do a generic query like `curl -X POST --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":99999}'`, see https://eth.wiki/json-rpc/API#eth_blocknumber or `eth_protocolVersion`, but not using curl but Go's internal http requests.

Reply:
Good idea, i will look into it once i fixed all the other things.

Reply:
I added a retry loop for the dial.